_posts/2017-03-24-sequence-labeling-on-a-structured-data.md

```diff
@@ -128,7 +134,9 @@ Consider that this one:
 is the **same** XML document as before, just a different serialized form of it.
 
 We can hope that given a lot of data system can eventually learn that order of attributes does not matter and that
-`cinqui&#232;me` represents the same word as `cinquième`. But it is obviously a huge and artificial complication.
+`cinqui&#232;me` represents the same word as `cinquième`. Alternatively, we may want to normalize the serialized text
```

Comment:
This will cause major problem when using lexicon lookup or word embedding lookup as a feature.